sdk/servicebus/Azure.Messaging.ServiceBus/src/Diagnostics/ServiceBusEventSource.cs

```diff
@@ -23,7 +23,7 @@ namespace Azure.Messaging.ServiceBus.Diagnostics
     ///   the CompleteEvent.Id must be exactly StartEvent.Id + 1.
     /// </remarks>
     [EventSource(Name = EventSourceName)]
-    internal class ServiceBusEventSource : EventSource
+    internal class ServiceBusEventSource : AzureEventSource
```

Comment:
How do we ensure that AzureEventSource is always used going forward?

Reply:
Change guidelines to start with. Maybe something in the test listener that validates.

Reply:
/cc @annelo-msft regarding guideline updates

Reply:
I'll do it.